### ISSUE TITLE
任何时候下拉超过距离都会触发onRefresh函数

### ISSUE THREAD

**Issue by wangmh1990** [lakb248/vue-pull-refresh#7]
比如我滚动了1000px的距离，此时向下滑动，也会触发onRefresh函数，应该还需要判断滚动页面是否在顶部。尝试了下，不知道是不是我页面布局有点乱的原因，获取的scrollTop都是0

**Comment by lakb248** [comment]
有示例吗

**Comment by wangmh1990** [comment]
https://lakb248.github.io/vue-pull-refresh/
你这个demo你把.content的height变成5000px，然后滚动到底部，在onfresh函数那打上断点，然后下滑，就可以复现(滑动的距离超过你设置的那个高度)

**Comment by pingshunhuangalex** [comment]
@lakb248 Can confirm this bug. Basically scrolling up a reasonably long page will keep triggering the refresh. Please add a condition to check if we are on the top of the page, otherwise, this useful component may render useless in this case. Thanks.

**Comment by lakb248** [comment]
Sorry, I will fix it this weekend

**Comment by lakb248** [comment]
@pingshunhuangalex @wangmh1990 bug fixed

**Comment by pingshunhuangalex** [comment]
Thanks mate, will test it out tomorrow, cheers

**Comment by pingshunhuangalex** [comment]
Hi @lakb248 
I just tested things out, it works to some degree.

Yes, it will only be triggered when it's at the top of the page. However, if you scroll (run your fingers through the screen) quickly, it'll also get triggered by the scrolling momentum if the page reaches the top.

An expected behaviour would be when user's finger leaves the screen (`touchend`), the plugin shouldn't be triggered even at the top of the screen. At the moment, I checked if I'm on the top of the page in the `onRefresh` method which works well too, but it would still be nice to take this case into the consideration. Thanks.
